omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -47,7 +47,91 @@
     <script type="text/javascript">
 
         if (jQuery) {
-            jq(document).ready(function () {
+        jq(document).ready(function () {
```

Comment:
@solemabrothers Indent the Javascript, we should look into not having to duplicate the code for medicines and other medications